packages/patternfly-4/content/design-guidelines/usage-and-behavior/text-input/design.md

```diff
@@ -0,0 +1,42 @@
+---
+path: "/design-guidelines/usage-and-behavior/text-input"
+---
+# Text input
+Use text input to gather free-form text from a user.
+
+## Elements
+
+![Elements](img/input-elements3.png)
+
+1. Label
+2. Input field
+3. Help
+4. Errors and validation
```

Comment:
Change to? "Inline Error: Errors resulting from validation of field data should be placed below the field." 

Link to more details below?